src/mono/wasm/templates/templates/console/README.md

```diff
@@ -12,10 +12,10 @@ After building the application, the result is in the `bin/$(Configuration)/net7.
 
 ## Run
 
-You can build the applcation from Visual Studio or by dotnet cli
+You can build the application from Visual Studio or by dotnet cli
 
 ```
-dotnet run -c Debug/Release -r browser-wasm -h=nodejs
+dotnet run -c Debug/Release -r browser-wasm --h=node
```

Comment:
Since we have `<RuntimeIdentifier>browser-wasm</RuntimeIdentifier>` and `runtimeconfig.template.json` maybe just `dotnet run` should be enough ?

Reply:
I think so.
I'll remove RuntimeIdentifier(-r) settings for the build arguments as well, leaving Configuration settings.